Projects/UOContent/Items/Skill Items/Magical/Misc/RecallRune.cs

```diff
@@ -268,40 +268,38 @@ public override void OnSingleClick(Mobile from)
                     LabelTo(
                         from,
                         House != null ? 1063260 : 1063259, // ~1_val~ (Tokuno Islands)[(House)]
-                        string.Format(RuneFormat, desc)
+                        $"{RuneFormat} {desc}"
                     );
                 }
                 else if (m_TargetMap == Map.Malas)
                 {
                     LabelTo(
                         from,
                         House != null ? 1062454 : 1060804, // ~1_val~ (Malas)[(House)]
-                        string.Format(RuneFormat, desc)
+                        $"{RuneFormat} {desc}"
                     );
                 }
                 else if (m_TargetMap == Map.Felucca)
                 {
                     LabelTo(
                         from,
                         House != null ? 1062452 : 1060805, // ~1_val~ (Felucca)[(House)]
-                        string.Format(RuneFormat, desc)
+                        $"{RuneFormat} {desc}"
                     );
                 }
                 else if (m_TargetMap == Map.Trammel)
                 {
                     LabelTo(
                         from,
                         House != null ? 1062453 : 1060806, // ~1_val~ (Trammel)[(House)]
-                        string.Format(RuneFormat, desc)
+                        $"{RuneFormat} {desc}"
                     );
                 }
                 else
                 {
                     LabelTo(
                         from,
-                        House != null ? "{0} ({1})(House)" : "{0} ({1})",
-                        string.Format(RuneFormat, desc),
-                        m_TargetMap
+                        $"{string.Format(RuneFormat, desc)} ({m_TargetMap}){(House != null ? "(House)" : string.Empty)}"
```

Comment:
The variable RuneFormat is not a string formatter anymore. Let's flatten this out into the parent interpolation.

Reply:
Done.